SORA Card KYC tutorial ahead of launch (#122)
sora-xor/sora-docs: src/sora-card-kyc-tutorial.md

---
title: How to Sign Up for SORA Card - A Comprehensive Guide | SORA Docs
head:
  - - meta
    - name: description
      content: Discover a step-by-step walkthrough guide on how to sign up for a SORA Card, undergo the KYC process, and ensure ease of mind. Learn about the channels, documentation, and seamless registration process.

  - - meta
    - name: keywords
      content: SORA Card sign up, KYC process, SORA Card application channels, required documents for SORA Card, registration and KYC flow, SORA Wallet mobile app, XOR purchase, seamless sign up process
---

> # **How to Sign Up for SORA Card, a Walkthrough**

Reviewer: ```suggestion
# How to Sign Up for SORA Card, a Walkthrough
```

> **In this article, you will find a walkthrough to sign up for SORA Card and undergo the KYC process.**

Reviewer: ```suggestion
```
there is a similar "in this article you will find" line below so I think you can remove this

Although signing up and undergoing KYC for SORA Card is straightforward, this article will help you resolve any questions you might have.

> **Important to note that only EU residents are currently eligible to go through the KYC process. This will change shortly.**

Reviewer: ```suggestion
::: info
Important to note that only EU residents are currently eligible to go through the KYC process. This will change shortly.
:::
```

Author: heh I also wanted to do this :D

This process only applies to consumer accounts.

**In this article, you will find:**

1. The channels and options to apply for SORA Card
2. The documents and information you should prepare for a smooth KYC process
3. The registration and KYC flow

> You can read more about where to sign up and the pre-requirements in the article that announced SORA Card signups. We have summarised it again below.

Reviewer: `in the article that announced SORA Card signups` - link?

## 1. Channels and options to apply for SORA Card

There are two channels to apply for a SORA Card:

1. On the [polkaswap.io](http://polkaswap.io/) dApp
2. Through the SORA Wallet mobile app (available on [Google Play](https://play.google.com/store/apps/details?id=jp.co.soramitsu.sora&hl=en&gl=US&pli=1) and the [Apple App Store](https://play.google.com/store/apps/details?id=jp.co.soramitsu.sora&hl=en&gl=US&pli=1))

> - **_If you just downloaded SORA Wallet, store your seed phrase safely._**

Reviewer: ```suggestion
::: tip
If you just downloaded SORA Wallet, store your seed phrase safely.
:::
```

And the three options to apply for SORA Card, along with the fees for each option, are:

1. **If you already have more than €100 worth of XOR in your account, staked, or in a pool where you provide liquidity**: the application with your SORA address is free;
2. **If you want to purchase XOR to have the €100 balance for a free order**, you can buy XOR worth up to €700 within the app with only soft KYC required (details below);
3. **If you do not have €100 worth of XOR and don’t want to purchase it**, there will be a fee of €20 to apply for KYC and the SORA Card available in the second phase. This payment can be made with any debit/credit card\*.

> _You can use a credit/debit card from Mastercard®, Visa®, Discover®, Diners Club® or JCB® for the purchase._

Reviewer: ```suggestion
3. **If you do not have €100 worth of XOR and don’t want to purchase it**, there will be a fee of €20 to apply for KYC and the SORA Card available in the second phase. This payment can be made with any debit/credit card from Mastercard®, Visa®, Discover®, Diners Club® or JCB®.
```

> ## 2. Prepare this documentation and the following personal data;

Reviewer: ```suggestion
## 2. Required documents and personal data
```

A. In case you already have €100 worth of XOR on the account, **continue to B**. To **buy up to €100 worth of XOR** with a soft KYC, you will need to provide your

- Name
- Home address
- Email address
- Nationality

_Unfortunately, you cannot access this service if your nationality is from [one of these countries](https://x1ex.com/en/countries). You can still purchase XOR via alternative ways._

> Current alternatives are: swapping on [polkaswap.io](http://polkaswap.io/) and Uniswap. As well as buying from [Gate.io](http://gate.io/) and then bridging to your SORA account on [Polkaswap.io](http://polkaswap.io/).

Reviewer: ```suggestion
::: info
Unfortunately, you cannot access this service if you are from [one of these countries](https://x1ex.com/en/countries). You can still purchase XOR via alternative ways.

Current alternatives are: swapping on [polkaswap.io](http://polkaswap.io/) and Uniswap. As well as buying from [Gate.io](http://gate.io/) and then bridging to your SORA account on [Polkaswap.io](http://polkaswap.io/).
:::
```

B. To undergo **KYC for SORA Card,** you need to prepare the following documents and information:

- **Your mobile phone number**. This will be verified with a code sent to this number.
- **Full name** as written on your official documents
- **Personal email address**. A verification link will be sent.
- **Photo of your ID** document
- **Photo of proof of address**: an official document (utility bill, bank statement, government statement or correspondence) that contains your full name and address and is not older than 3 months.
- **Address of residency**

> - Unfortunately, you can not access the service if you reside in one of the [countries on this list](https://soracard.com/blacklist/).

Reviewer: ```suggestion
   ::: info
   Unfortunately, you can not access the service if you reside in one of the [countries on this list](https://soracard.com/blacklist/).
   :::
```

- **A Selfie** will be taken during the KYC process.

## 3. Registration and KYC Flow

Below you will find the registration flow that initiates the KYC process.
The KYC process is done through the integration of SumSub software, which will only disclose your data to the banking partner reviewing your application.

![](/.gitbook/assets/sc-Details.png)

> To start the KYC process, input your phone number, then request the verification code. Be sure to input your phone number in the international format (starting with the country code).

Reviewer: please make these numbered steps

![](/.gitbook/assets/sc-Enter-Phone.png)

Enter the code as received in the SMS.

![](/.gitbook/assets/sc-Verify-Phone.png)

After verifying your phone number, input your full name as it appears on the official identification document you use for the KYC process.

![](/.gitbook/assets/sc-Enter-Name.png)

> Input your email address and click the magic link received in your inbox. After completing the verification, return to the app's KYC screen.

Reviewer: ```suggestion
Input your email address and click the verification link received in your inbox. After completing the verification, return to the app's KYC screen.
```

![](/.gitbook/assets/sc-Enter-Email.png)

Every applicant has a credit of 2 KYC attempts included in the application. This applies to all applicants, including applicants with €100 worth of XOR on their SORA account and applicants who order SORA Card by paying €20.

> To avoid an extra cost of €3.80 per additional attempt, please ensure the following;

Reviewer: ```suggestion
To avoid an extra cost of €3.80 per additional attempt, please ensure the following:
```

- Your identification document is valid and legible
- Your selfie is clear and following the guidance provided
- Your proof of address document is valid and legible
- The personal information form is filled out honestly and accurately.

In any case, **the bank retains the right to the final decision over your application**.

![](/.gitbook/assets/sc-KYC-Summary.png)

> When the verification of your application is successful, your IBAN will be enabled shortly (ETA 1 week), and your SORA Card will be sent to you once SORA Card Phase 2 is live. (you will be emailed about the status once the physical card shipment becomes available).

Reviewer: ```suggestion
When the verification of your application is successful, your IBAN will be enabled shortly (ETA 1 week), and your SORA Card will be sent to you once SORA Card Phase 2 is live. You will be emailed about the status once the physical card shipment becomes available.
```

![](/.gitbook/assets/sc-Verification-successful.png)

If your application is rejected, you will see this screen with the reason for the rejection.

![](/.gitbook/assets/sc-Verification-rejected.png)

When you see this screen, you have used up your complementary KYC attempts, and additional attempts come at €3.80 per attempt. Additional paid KYC will be available from a future release onward.

> If you have any issues or questions, you can contact our community support on [Telegram](https://t.me/SORAhappiness) or through the Support button in the app

Reviewer: ```suggestion
If you have any issues or questions, you can contact our community support on [Telegram](https://t.me/SORAhappiness) or through the Support button in the app.
.
```

## Next steps

The SORA Card team will continue to provide regular updates to upgrade and improve the product and user experience based on your feedback. Your IBAN will be enabled and activated shortly after your KYC has been approved. This will allow euro transactions, such as SEPA transfers, to and from your account.

> In the next major release, you will receive a virtual and physical card, and the crypto on/offramp to euro, as well as an exchange, will be enabled If you have any further questions or feedback, please reach out on the [main SORA](https://t.me/sora_xor) and [SORA Happiness](https://t.me/SORAhappiness) chats

Reviewer: ```suggestion
In the next major release, you will receive a virtual and physical card, and the crypto on/offramp to euro, as well as an exchange, will be enabled. If you have any further questions or feedback, please reach out on the [main SORA](https://t.me/sora_xor) and [SORA Happiness](https://t.me/SORAhappiness) chats.
```
